### DISCUSSION TITLE
Error: status 499

### DISCUSSION THREAD

**Discussion by vincent-thomas** [community/community#48605] [Codespaces]
### Select Topic Area

Bug

### Body

My github Codespaces randomly crashes alot with the Remote Extension host and pty and even the connection to codespaces crashes! Heres an image example

![](https://i.imgur.com/Y6maNRh.png)

It keeps crashing and i don't really know why. Maybe the team behind codespaces could provide some help?

**Comment by dmilburn** [comment]
Hi from the codespaces team! I'm sorry for the inconvenience. Could you share the name of the codespace that's crashing? You can find this by navigating to github.com/codespaces and copying the link you would click to open the codespace.

**Reply by sanchaiyo** [reply]
@dmilburn 
I also get this issue, here is my codespaces https://github.com/codespaces/sanchaiyo-congenial-happiness-9jpjq6v57xpf7qpp?editor=vscode

**Reply by ManuGS2** [reply]
I'm having the same issue. It started about 3 days ago.
Any update on this?

**Reply by bhildebay** [reply]
Same here

**Reply by zenhorace** [reply]
also having this issue :(
